ecosystem/sep-0024.md

```diff
@@ -190,6 +191,24 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Protocol
+14/15](https://www.stellar.org/blog/protocol-14-improvements?locale=en) now allows anchors to
+complete payments using the [ClaimableBalance
+functionality](../core/cap-0023.md) and the user can collect the funds at any
+time.
+
+This is an optional, yet recommended, feature that will no longer need to wait
+for the `Wallet` or the user to establish a
+trust line for the `Account` for the given asset.
```

Comment:
```suggestion
[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allow for payments to accounts that do not have a trustline for the asset sent. 

Anchors can optionally use this functionality to send SEP-24 deposit transactions to Stellar accounts that don't yet have a trustline to the asset they have requested.

Wallets should poll for their users' received Claimable Balances so that users can "claim" them. Claiming a Claimable Balance adds a trustline to the account for the asset claimed and adds the amount transacted.
```